accounts/models.py

```diff
@@ -411,3 +411,11 @@ def __unicode__(self):
 class UserEmailSetting(models.Model):
     user = models.ForeignKey(User, related_name="email_settings")
     email_type = models.ForeignKey(EmailPreferenceType)
+
+
+class OldUsername(models.Model):
```

Comment:
Shouldn't we be auto-filling these if the username gets changed - for users who have sounds this only happens through the admin, btw?

Reply:
yes, we only change this using the admin. I added the auto-filling :+1: